tests/gh-actions/install_multi_tenancy.sh

```diff
@@ -10,3 +10,6 @@ kustomize build common/kubeflow-roles/base | kubectl apply -f -
 
 echo "Installing Multitenancy Network policies"
 kustomize build common/networkpolicies/base | kubectl apply -f -
+
+echo "Enabling restricted levels Pod Security Standards to kubeflow namespace"
+kubectl patch namespace kubeflow --patch-file contrib/security/PSS/static/restricted/patches/kubeflow-labels.yaml
```

Comment:
Check if the namespace if it exists, if yes patch it. Start with baseline now and deal with restricted later in follow up PRs

Reply:
The patch for the securitycontext of the profile controller belongs to /contrib/security as well and needs to be applied here.